docker-compose.yml

```diff
@@ -26,8 +26,5 @@ services:
       - PERSISTENCE=1
       - DOCKER_HOST=unix:///var/run/docker.sock
       - HOSTNAME_EXTERNAL=localhost
-    volumes:
-      - "/var/run/docker.sock:/var/run/docker.sock"
-      - "$PWD/src/test/resources/localstack/setup-aws.sh:/etc/localstack/init/ready.d/init-aws.sh"
```

Comment:
are you sure about this? I think it sets up the S3 bucket required for large message processing